mobile/src/main.jsx

```diff
@@ -39,10 +39,7 @@ document.addEventListener('DOMContentLoaded', () => {
   localforage.getItem('state').then(state => {
     if (state) {
       store.dispatch({ type: SET_STATE, state })
-      cozy.init({
-        cozyURL: state.serverUrl,
-        offline: {doctypes: ['io.cozy.files'], timer: 15}
-      })
+      init(state.serverUrl)
```

Comment:
I checked with @aenario:

For third party application, cozy-client must be init with a oauth block. So I add this oauth block in the cozy-helper `init` function. No need for `onRegister` callback as I explain in https://github.com/cozy/cozy-files-v3/pull/83/commits/b90e252120d709fd4da48075c6182461e94116c6.